swatch-contracts/src/main/java/com/redhat/swatch/contract/service/CapacityReconciliationService.java

```diff
@@ -78,8 +78,18 @@ public void reconcileCapacityForOffering(String sku, int offset, int limit) {
     subscriptions.forEach(this::reconcileCapacityForSubscription);
     if (subscriptions.size() >= limit) {
       offset = offset + limit;
-      reconcileCapacityByOfferingEmitter.send(
-          ReconcileCapacityByOfferingTask.builder().sku(sku).offset(offset).limit(limit).build());
+      reconcileCapacityByOfferingEmitter
+          .send(
```

Comment:
There is an easier way to do this by changing the `Emitter` to use `MutinyEmitter` which has the method `sendAndAwait` that does the wait already. 
This is just a minor suggestion, tho we also use the `MutinyEmitter` in other producers in our project.